src/main/java/org/scalameter/annotations/benchmark.java

```diff
@@ -0,0 +1,31 @@
+package org.scalameter.annotations;
```

Comment:
You're right - although not many people are using it, it might be a better idea to first deprecate it.
I fear that, if we want to later move the annotations from the `annotation` subpackage back to `japi`, we need to undergo another deprecation cycle.